### PR TITLE
Main page title: Short bord red underline - follow the title alignement

### PR DESCRIPTION
**Pull request by polmih**
If the main title is aligned to right (dir="rtl") the bottom red underline will follow the same alignement.

### CODE REVIEW

**Comment by Garneauma** [comment]
Pre-approved upon successful review.

File: sites/main-page-title/_base.scss [hunk 1]
```diff
@@ -9,6 +9,16 @@ h1#wb-cont, hgroup#wb-cont h1 {
 	border-top-width: 0;
 }
 
+h1#wb-cont[dir="rtl"], hgroup#wb-cont h1[dir="rtl"] {
```
Comment (duboisp) [inline sites/main-page-title/_base.scss:12]: Missing an example with `hgroup`.

Should we expand the selector? like:
* `[dir="rtl"] h1#wb-cont`
* Not sure about this one, I suggest to wait for a use case before:  `hgroup[dir="rtl"]#wb-cont h1`

Comment (polmih) [reply sites/main-page-title/_base.scss]: example added. Thanks

**Review by duboisp — CHANGES_REQUESTED** [review]
Missing an example where:
```
<div dir=rtl>
   <hgroup id=wb-cont>
     <h1>Title 1</h1>
     <p>Group</p>
   </hgroup>
</div>
```
To represent situation when "rtl" is set on a parent element like "main" or "html"

**Comment by polmih** [comment]
> Missing an example where:
> 
> ```
> <div dir=rtl>
>    <hgroup id=wb-cont>
>      <h1>Title 1</h1>
>      <p>Group</p>
>    </hgroup>
> </div>
> ```
> 
> To represent situation when "rtl" is set on a parent element like "main" or "html"

The missing example is now added. Thanks

**Comment by Garneauma** [comment]
@polmih Could you please fix the build issue?

File: sites/main-page-title/_base.scss [hunk 2]
```diff
@@ -9,6 +9,16 @@ h1#wb-cont, hgroup#wb-cont h1 {
 	border-top-width: 0;
 }
 
+h1#wb-cont[dir="rtl"], hgroup#wb-cont[dir="rtl"] h1, [dir="rtl"] h1#wb-cont {
```
Comment (duboisp) [inline sites/main-page-title/_base.scss:12]: Missing this selector

```suggestion
h1#wb-cont[dir="rtl"], hgroup#wb-cont[dir="rtl"] h1, [dir="rtl"] h1#wb-cont, [dir="rtl"] hgroup#wb-cont h1 {
```

File: sites/main-page-title/main-page-title-div-stacked-align-right-en.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+altLangPage: "main-page-title-div-stacked-align-right-fr.html"
+dateModified: "2024-09-26"
+language: "en"
+pageclass: "wb-prettify all-pre"
+layout: without-h1
+index_json: index.json-ld
+---
+<div dir="rtl">
+  <hgroup id="wb-cont" dir="rtl">
```
Comment (duboisp) [inline sites/main-page-title/main-page-title-div-stacked-align-right-en.md:10]:
```suggestion
  <hgroup id="wb-cont">
```

File: sites/main-page-title/main-page-title-div-stacked-align-right-fr.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+altLangPage: "main-page-title-div-stacked-align-right-en.html"
+dateModified: "2024-09-26"
+language: "fr"
+pageclass: "wb-prettify all-pre"
+layout: without-h1
+index_json: index.json-ld
+---
+<div dir="rtl">
+  <hgroup id="wb-cont" dir="rtl">
```
Comment (duboisp) [inline sites/main-page-title/main-page-title-div-stacked-align-right-fr.md:10]:
```suggestion
  <hgroup id="wb-cont">
```

**Review by duboisp — CHANGES_REQUESTED** [review]
Quick code review, see my comments. I didn't tested locally.

File: sites/main-page-title/_base.scss (hunk 2 above)
Comment (polmih) [reply sites/main-page-title/_base.scss]: Added. Thanks

File: sites/main-page-title/main-page-title-div-stacked-align-right-fr.md (hunk 4 above)
Comment (polmih) [reply sites/main-page-title/main-page-title-div-stacked-align-right-fr.md]: Fixed. thanks

File: sites/main-page-title/main-page-title-div-stacked-align-right-en.md (hunk 3 above)
Comment (polmih) [reply sites/main-page-title/main-page-title-div-stacked-align-right-en.md]: fixed.thanks

File: sites/main-page-title/index.json-ld [hunk 5]
```diff
@@ -354,6 +394,12 @@
 			],
 			"predecessor": "_:iteration_mpt_2"
 		},
+		{
+			"@id": "_:iteration_mpt_4",
+			"name": "Main page title aligned on the right",
+			"date": "2024-09",
+			"detectableBy": "<h1 property='name' id='wb-cont' dir='rtl'></h1>"
```
Comment (Garneauma) [inline sites/main-page-title/index.json-ld:401]:
```suggestion
			"detectableBy": "h1#wb-cont[dir='rtl'], [dir='rtl'] h1#wb-cont",
			"additions": [
				"Added left to right support"
			]
```

File: sites/main-page-title/index.json-ld [hunk 6]
```diff
@@ -354,6 +394,12 @@
...
+		},
```
Comment (Garneauma) [inline sites/main-page-title/index.json-ld:402]:
```suggestion
		},
		{
			"@id": "_:iteration_smpt_2",
			"name": "Stacked main page title - Iteration 2 (LTR support)",
			"date": "2024-10",
			"detectableBy": "hgroup > h1#wb-cont[dir='ltr'], [dir='ltr'] hgroup > h1#wb-cont",
			"additions": [
				"Added left to right support"
			]
		}
```

**Review by Garneauma — CHANGES_REQUESTED** [review]
- [x] Version should be changed to 1.2 (index.json-ld line 20)
- [x] In "history" of both variations, add a history item for the addition of the LTR support (line 132 and line 164)
- [x] Add the newly created working examples links to the list on each variation accordingly.
- [ ] Please add a breadcrumb item that refers to the documentation on each working example. I couldn't do so on the main example as it is shared for many components, but the new ones are specific to this component.

File: sites/main-page-title/index.json-ld (hunk 5 above)
Comment (polmih) [reply sites/main-page-title/index.json-ld]: Fixed. thanks

File: sites/main-page-title/index.json-ld (hunk 6 above)
Comment (polmih) [reply sites/main-page-title/index.json-ld:429]: Added. Thanks

File: sites/main-page-title/index.json-ld [hunk 7]
```diff
@@ -354,11 +418,29 @@
...
 			"predecessor": "_:iteration_mpt_2"
 		},
+		{
+			"@id": "_:iteration_mpt_4",
+			"name": "Main page title aligned on the right",
+			"date": "2024-09",
+			"detectableBy": "h1#wb-cont[dir='rtl'], [dir='rtl'] h1#wb-cont",
+			"additions": [
+				"Added right to left support"
+			]
```
Comment (Garneauma) [inline sites/main-page-title/index.json-ld:428]: Please also add `"successor": "_:iteration_mpt_4"` to the Iteration 3.

```suggestion
			],
			"predecessor": "_:iteration_mpt_3"
```

File: sites/main-page-title/index.json-ld [hunk 8]
```diff
@@ -354,11 +418,29 @@
...
 			"detectableBy": "hgroup > h1#wb-cont"
+		},
+		{
+			"@id": "_:iteration_smpt_2",
+			"name": "Stacked main page title - Iteration 2 (LTR support)",
+			"date": "2024-10",
+			"detectableBy": "hgroup > h1#wb-cont[dir='ltr'], [dir='ltr'] hgroup > h1#wb-cont",
+			"additions": [
+				"Added right to left support"
+			]
```
Comment (Garneauma) [inline sites/main-page-title/index.json-ld:443]: Please also add `"successor": "_:iteration_smpt_2"` to the Iteration 1.
```suggestion
			],
			"predecessor": "_:iteration_smpt_1"
```

File: sites/main-page-title/main-page-title-align-right-en.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
+---
+altLangPage: "main-page-title-align-right-fr.html"
+dateModified: "2024-09-26"
+language: "en"
+pageclass: "wb-prettify all-pre"
+layout: without-h1
+title: "Main page title aligned on the right"
+index_json: index.json-ld
+breadcrumbs: [
+		{ "title": "Titre principal - Documentation", "link": "sites/main-page-title/main-page-title-en.html" }
```
Comment (Garneauma) [inline sites/main-page-title/main-page-title-align-right-en.md:10]: Please adjust the title of the breadcrumb link to be in English.

File: sites/main-page-title/main-page-title-div-align-right-en.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
+---
+altLangPage: "main-page-title-div-align-right-fr.html"
+dateModified: "2024-09-26"
+language: "en"
+pageclass: "wb-prettify all-pre"
+layout: without-h1
+title: "Main page title in a div aligned on the right"
+index_json: index.json-ld
+breadcrumbs: [
+		{ "title": "Titre principal - Documentation", "link": "sites/main-page-title/main-page-title-en.html" }
```
Comment (Garneauma) [inline sites/main-page-title/main-page-title-div-align-right-en.md:10]: Please adjust the title of the breadcrumb link to be in English.

File: sites/main-page-title/main-page-title-div-stacked-align-right-en.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
+---
+altLangPage: "main-page-title-div-stacked-align-right-fr.html"
+dateModified: "2024-09-26"
+language: "en"
+pageclass: "wb-prettify all-pre"
+layout: without-h1
+title: "Stacked main page title in a div aligned on the right"
+index_json: index.json-ld
+breadcrumbs: [
+		{ "title": "Titre principal - Documentation", "link": "sites/main-page-title/main-page-title-en.html" }
```
Comment (Garneauma) [inline sites/main-page-title/main-page-title-div-stacked-align-right-en.md:10]: Please adjust the title of the breadcrumb link to be in English.

File: sites/main-page-title/main-page-title-stacked-align-right-en.md [hunk 12]
```diff
@@ -0,0 +1,22 @@
+---
+altLangPage: "main-page-title-stacked-align-right-fr.html"
+dateModified: "2024-09-26"
+language: "en"
+pageclass: "wb-prettify all-pre"
+layout: without-h1
+title: "Stacked main page title aligned on the right"
+index_json: index.json-ld
+breadcrumbs: [
+		{ "title": "Titre principal - Documentation", "link": "sites/main-page-title/main-page-title-en.html" }
```
Comment (Garneauma) [inline sites/main-page-title/main-page-title-stacked-align-right-en.md:10]: Please adjust the title of the breadcrumb link to be in English.

**Review by Garneauma — CHANGES_REQUESTED** [review]
- [x] index.json-ld - line 161: change to smpt_2
- [x] index.json-ld - line 239: change to mpt_4
- [x] index.json-ld - line 275: change to smpt_2
- [ ] index.json-ld - line 311: change to smpt_2
- [ ] index.json-ld - line 350: change to mpt_4
- [ ] index.json-ld - line 373: change to smpt_2
- [ ] implement other changes requested in comments

File: sites/main-page-title/index.json-ld (hunk 7 above)
Comment (polmih) [reply sites/main-page-title/index.json-ld]: added. Thanks

File: sites/main-page-title/index.json-ld (hunk 8 above)
Comment (polmih) [reply sites/main-page-title/index.json-ld]: Added. Thanks

File: sites/main-page-title/main-page-title-align-right-en.md (hunk 9 above)
Comment (polmih) [reply sites/main-page-title/main-page-title-align-right-en.md]: Done. Thanks

File: sites/main-page-title/main-page-title-div-align-right-en.md (hunk 10 above)
Comment (polmih) [reply sites/main-page-title/main-page-title-div-align-right-en.md]: Done. Thanks

File: sites/main-page-title/main-page-title-div-stacked-align-right-en.md (hunk 11 above)
Comment (polmih) [reply sites/main-page-title/main-page-title-div-stacked-align-right-en.md]: Done. Thanks

File: sites/main-page-title/main-page-title-stacked-align-right-en.md (hunk 12 above)
Comment (polmih) [reply sites/main-page-title/main-page-title-stacked-align-right-en.md]: Done. Thanks

**Pull request merged**